tests/validators/test.chromemanifest.js

```diff
@@ -0,0 +1,55 @@
+import { Readable } from 'stream';
+import { VALIDATION_WARNING } from 'const';
+
+import ChromeManifestScanner from 'validators/chromemanifest';
+
+
+describe('ChromeManifestScanner', () => {
+
+  it('should detect dangerous categories', () => {
+    var rstream = new Readable();
+    rstream.push('content  necko   jar:comm.jar!/content/necko/\n');
+    rstream.push('category JavaScript-DOM-class foo bar\n');
+    rstream.push(null);
+    var cmScanner = new ChromeManifestScanner(rstream, 'chrome.manifest');
+    return cmScanner.scan()
+      .then((messages) => {
+        assert.equal(messages.length, 1);
+        assert.equal(messages[0].code, 'DANGEROUS_CATEGORY');
```

Comment:
I've been importing `messages` and doing `messages.DANGEROUS_CATEGORY.code` to reduce having to change things if things change... but now I wonder if that's just more verbose as we want the codes to be the same as the constant name.

Maybe I should go through tests I've written and change mine to this way?